### DISCUSSION TITLE
MaterialEditText 的 floatingLabel 不想要半透明怎么办？

### DISCUSSION THREAD

**Discussion by kvgnt** [xuexiangjys/XUI#128] [Q&A]
floatingLabel 的半透明很影响操作体验，对一些弱视群体不友善，请问如何把半透明效果去掉。

**Comment by xuexiangjys** [comment]
> floatingLabel 的半透明很影响操作体验，对一些弱视群体不友善，请问如何把半透明效果去掉。

met_floatingLabel设置成highlight

**Reply by kvgnt** [reply]
met_floatingLabel设置成highlight 只是让标题在获取焦点时有变化，并不能改变没获取焦点时的样式。
[图片示例](https://imgtu.com/i/oz5DR1)
获取焦点跟失去焦点，区别很大，失去焦点时色弱很难看到标题。

normal：让标题在获取焦点时不变色，一直都是半透明
highlight：让标题获取焦点时变色，并恢复不透明。但是无焦点情况下还是半透明，无法看清。

**Reply by xuexiangjys** [reply]
这个控件目前还没有除你之外的人有反馈这个问题,我理解是小众需求,你自己修改源码吧

**Reply by kvgnt** [reply]
> 这个控件目前还没有除你之外的人有反馈这个问题,我理解是小众需求,你自己修改源码吧

我之前看过源码，知道是去改那行（没找到配置开关才来问），但不想对外部库进行修改，已经在不改代码的情况下增加一个textview替换floatingLabel 二次封装解决了。

这里提出来是因为它确实是个UI问题。这个需求不是程序员需要，是使用者需要。老年人(弱视群体)确实是手机APP的小众，但是人是会变老的（我只是想的比别人长远点），这个需求只会越来越多。多个开关对你来说只是多写两行代码，但是对用户来说，是可以立即改善体验。望采纳。
